specs/Renting/P2PCarRental/doc/spec.md

```diff
@@ -0,0 +1,75 @@
+# P2PCarRental
+
+## Description
+
+This entity contains a description of a peer to peer rental car.
```

Comment:
this is just a vehicle and we already have a Vehicle model ...

Reply:
I checked the existing vehicle model and it did not seem to have all the stuff we need here. I'll check again to see how we can use that existing vehicle model. ANy suggestions?